docs/src/getting-started/override.rst

```diff
@@ -52,12 +52,13 @@ possibility. The changes above can be achieved by typing:
 .. code-block:: bash
 
    mtt train options.yaml \
-      -r architecture.model.soap.cutoff=7.0 architecture.training.num_epochs=200
+      -r architecture.model.soap.cutoff=7.0,architecture.training.num_epochs=200
```

Comment:
Maybe we should rather let people set the `-r` argument multiple times.

```bash
mtt train options.yaml \
    -r architecture.model.soap.cutoff=7.0 \
    -r architecture.training.num_epochs=200 \
```

argparse supports this using the `action='append'` option.